libraries/availableModules.js

```diff
@@ -0,0 +1,33 @@
+let nodeTypes = {};
+let hardwareInterfaces = {};
+let blockTypes = {};
+
+const setNodes = function(nodeTypeModules) {
```

Comment:
These declarations could be less redundant as
```javascript
exports.setNodes = function setNodes(nodeTypeModules) {
}
```

Technically the function name is also redundant and unnecessary but I think it might help in stack traces